### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by mm111222** [XunpengYi/Text-IF#20]
I would like to ask if the training part of the EMS Full dataset you provided  only comes from the training set of source MSRS,RoadScene and LLVIP?Did you use the source test sets of MSRS,RoadScene and LLVIP as your text free comparison experiment test set?
